tests/integration_tests/build/test_coverage.py

```diff
@@ -23,7 +23,7 @@
 # this contains the frequency while on AMD it does not.
 # Checkout the cpuid crate. In the future other
 # differences may appear.
-COVERAGE_DICT = {"Intel": 85.08, "AMD": 85.00}
+COVERAGE_DICT = {"Intel": 85.08, "AMD": 84.28}
```

Comment:
Is it possible to address this drop in coverage or is it code covered in integ tests now?

Reply:
The drop comes from the fact that the T2 and C3 code is not executed on AMD anymore. I don't see any way to address it. Only if we could mark this code as unreachable, but I don't know if it's possible.

Reply:
Since this code is platform specific, I think is fine to have this drop.